templates/user-research/index.html

```diff
@@ -0,0 +1,232 @@
+          </div>
+          <div class="col-6">
+            <p>Our 1:1 interview sessions are done remotely, and usually take 30-minutes to an hour. Typically you'll share your screen. Surveys only take a few minutes. Other tests may involve completing a task with a prototype and answering online questions.</p>
+          </div>
+        </div>
+        <hr class="p-rule">
+        <div class="row p-block">
+          <div class="col-3">
+            <p class="u-no-margin--bottom"><strong>How often will I be contacted?</strong></p>
+          </div>
+          <div class="col-6">
+            <p>Usually less than three times a year.</p>
+          </div>
+        </div>
+        <hr class="p-rule">
+        <div class="row p-block">
+          <div class="col-3">
+            <p class="u-no-margin--bottom"><strong>Will I receive something for participating?</strong></p>
+          </div>
+          <div class="col-6">
+            <p>When participating in some lengthy moderated sessions, such as 1-1 interviews or usability tests, we'll send you a thank you gift.</p>
+          </div>
+        </div>
+        <hr class="p-rule">
+        <div class="row p-block">
+          <div class="col-3">
+            <p class="u-no-margin--bottom"><strong>What will you do with my personal information?</strong></p>
+          </div>
+          <div class="col-6">
+            <p>Our data privacy policy defines how we collect <a href="/legal/data-privacy">the minimum amount</a> of personal information necessary to match you with suitable research studies.</p>
+          </div>
+        </div>
+        <hr class="p-rule">
+        <div class="row p-block">
+          <div class="col-3">
+            <p class="u-no-margin--bottom"><strong>Can I leave the group?</strong></p>
+          </div>
+          <div class="col-6">
+            <p>Absolutely. Just email <a href="mailto:userresearch@canonical.com">userresearch@canonical.com</a> and we'll delete your info from our database.</p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section is-deep">
+    <hr class="p-rule is-fixed-width">
+    <div class="row">
+      <div class="col-3 col-medium-2">
+        <p class="p-muted-heading">Talk to us</p>
+      </div>
+      <div class="col-9 col-medium-4">
+        <p>We want to hear about your user experiences &ndash; what's working well, and what's not. Get in touch to tell us what you think, so we can improve Canonical's products and services for everyone.</p>
+        <p><a href="https://discourse.ubuntu.com/t/user-research-programme/33805" class="p-button">User Research on Discourse</a></p>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section is-deep">
+    <hr class="p-rule is-fixed-width">
+    <div class="row">
+      <div class="col-3 col-medium-2">
+        <p class="p-muted-heading">Our <a href="/community/mission">mission</a></p>
```

Comment:
"From our mission" on the doc

Reply:
Go with the design on these, copydoc should have been updated